framework/wazuh/core/indexer/models/agent.py

```diff
@@ -52,7 +52,7 @@ class Status(str, Enum):
 class OS:
     """Agent operating system information."""
     name: str = None
-    platform: str = None
+    type: str = None
```

Comment:
This field is used in some unit tests, we should update them as well:

- https://github.com/wazuh/wazuh/blob/master/framework/wazuh/core/batcher/tests/test_client.py
- https://github.com/wazuh/wazuh/blob/master/apis/comms_api/core/test/test_events.py